model/database/registry.yaml

```diff
@@ -122,6 +122,19 @@ groups:
           For batch operations, if the individual operations are known to have the same
           stored procedure name then that stored procedure name SHOULD be used.
         examples: ["GetCustomer"]
+      - id: db.operation.parameter
+        type: template[string]
+        stability: development
+        brief: >
+          A database operation parameter, with `<key>` being the parameter name,
+          and the attribute value being a string representation of the parameter value.
+        note: >
+          If an operation parameter has no name and instead is referenced only by index,
+          then `<key>` SHOULD be the 0-based index.
+
```

Comment:
I think it's common for operation (not query) parameters to have name, so this is no longer needed?

```suggestion
```